src/app/datagrid/selection/selection.html

```diff
@@ -35,15 +35,35 @@
         <em *ngIf="selected.length == 0">No user selected.</em>
         <span class="username" *ngFor="let user of selected">{{user.name}}</span>
     </p>
+
+    <p class="card-text username-list">
+        Users to be added to group:
+        <em *ngIf="toAdd.length == 0">No user selected.</em>
+        <span class="username" *ngFor="let user of toAdd">{{user.name}}</span>
+    </p>
+
+    <p class="card-text username-list">
+        User to be deleted:
+        <em *ngIf="!toDelete">No user selected.</em>
+        <span class="username" *ngIf="toDelete">{{toDelete.name}}</span>
+    </p>
+
+    <p class="card-text username-list">
+        User to be edited:
+        <em *ngIf="!toEdit">No user selected.</em>
+        <span class="username" *ngIf="toEdit">{{toEdit.name}}</span>
+    </p>
 </div>
 
 <clr-datagrid [(clrDgSelected)]="selected">
     <clr-dg-action-bar>
-        <button type="submit" class="btn btn-link"><clr-icon shape="plus" size="16"></clr-icon> Add to group</button>
-        <button type="submit" class="btn btn-link"><clr-icon shape="close" size="16"></clr-icon> Delete</button>
-        <button type="submit" class="btn btn-link" *ngIf="selected.length == 1">
-            <clr-icon shape="pencil" size="16"></clr-icon> Edit
-        </button>
+        <div class="btn-group" role="group" aria-label="Available Actions">
+            <button type="button" class="btn btn-sm btn-secondary" (click)="onAdd()"><clr-icon shape="plus"></clr-icon> Add
+                to
+                group</button>
+            <button type="button" class="btn btn-sm btn-secondary"><clr-icon shape="close"></clr-icon> Delete</button>
+            <button type="button" class="btn btn-sm btn-secondary" *ngIf="selected.length == 1"><clr-icon shape="pencil"></clr-icon> Edit</button>
```

Comment:
I treated these demo components more like playground with incomplete details (as I am going to break up to several demo components for the website). But I updated it just for the sake of completeness.